From 08c7b20d90a1f49c3e27a05196dcd90531830fa7 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Thu, 28 Dec 2017 23:06:13 -0800
Subject: [PATCH 3007/3007] buildit: Fix CFLAGS for Leopard and Snow Leopard to
 include _LIBCPP_BUILDING_LIBRARY

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
(cherry picked from commit f2f970ff409e0c738a86f8bc20baf0310782518d)
---
 lib/buildit | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git libcxx-4.0.1.src/lib/buildit libcxx-4.0.1.src/lib/buildit
index d9b27d58e..60b2b1330 100755
--- libcxx-4.0.1.src/lib/buildit
+++ libcxx-4.0.1.src/lib/buildit
@@ -58,7 +58,7 @@ case $TRIPLE in
 
     if [ "$MACOSX_DEPLOYMENT_TARGET" = "10.5" ] || [ "$MACOSX_DEPLOYMENT_TARGET" = "10.6" ]
     then
-        EXTRA_FLAGS="-nostdinc++ -std=c++11 -U__STRICT_ANSI__"
+        EXTRA_FLAGS="${EXTRA_FLAGS} -U__STRICT_ANSI__"
         LDSHARED_FLAGS="-o libc++.1.dylib \
             -dynamiclib -nodefaultlibs \
             -current_version ${RC_ProjectSourceVersion} \
-- 
2.15.1

